src/admin/render_readmes.rs

```diff
@@ -203,18 +203,22 @@ fn render_pkg_readme<R: Read>(mut archive: Archive<R>, pkg_name: &str) -> Option
 
     let manifest: Manifest = {
         let path = format!("{}/Cargo.toml", pkg_name);
-        let contents = find_file_by_path(&mut entries, Path::new(&path), pkg_name);
+        let contents = find_file_by_path(&mut entries, Path::new(&path), pkg_name).unwrap();
         toml::from_str(&contents)
             .unwrap_or_else(|_| panic!("[{}] Syntax error in manifest file", pkg_name))
     };
 
     let rendered = {
-        let readme_path = manifest.package.readme.as_ref()?;
+        let readme_path = manifest
+            .package
+            .readme
+            .clone()
+            .unwrap_or("README.md".into());
```

Comment:
the docs mention:

> If no value is specified for this field, and a file named README.md, README.txt or README exists

Seems like only checking for `README.md` might not be sufficient? 🤔

Reply:
In order to handle this - I will first need to fix a bug where the script won't work if the archive entries are in a different order (such that it can look for multiple different paths in the archive). I think that work is sufficiently different, that I'll make it a separate PR.

I'd advocate for landing this as-is (as a step improvement), followed by the separate PRs for tar ordering and full support for different README types.